src/coins.c

```diff
@@ -15,7 +15,7 @@ REGISTER_COINS(
         .bip32Prefix = { HARDENED(44), HARDENED(165) },
         .addressPrimaryPrefix = "nano_",
         .addressSecondaryPrefix = "xrb_",
-        .addressDefaultPrefix = LIBN_SECONDARY_PREFIX,
+        .addressDefaultPrefix = "nano_",
```

Comment:
I'm directly mentioning `nano_` as I don't see the use for `LIBN_PRIMARY_PREFIX` here, but if for some reason `LIBN_PRIMARY_PREFIX` is better then I'd be happy to change it

Reply:
This is an enum (defined in [libn_types.h:55](https://github.com/roosmaa/ledger-app-nano/blob/master/src/libn_types.h#L55-L58)), using a string here wouldn't work. `LIBN_PRIMARY_PREFIX` would be the correct value here.

Reply:
Ah, okay. Sorry, I forgot to check that file